packages/@okta/vuepress-site/docs/guides/email-customization/customize-email-templates/index.md

```diff
@@ -59,23 +59,41 @@ Previously with EL syntax, you could reference the first name of the user by usi
 
 The previously available template variables are listed in [Customization Variables](https://help.okta.com/en/prod/Content/Topics/Settings/Settings_Email.htm).
```

Comment:
fixed, thanks!

Reply:
Sorry Albert, I was chatting with Maureen and she says we should be using aliases since these links could change.
Can you please change the link to this one?
https://help.okta.com/en/prod/okta_help_CSH.htm#ext_ref_email_variables